src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallGlobalOrToolPathCommand.cs

```diff
@@ -169,13 +169,17 @@ public override int Execute()
                 {
                     _environmentPathInstruction.PrintAddPathInstructionIfPathDoesNotExist();
                 }
-
+                if (_verbosity.IsQuiet())
+                {
+                    return 0;
+                }
                 _reporter.WriteLine(
-                    string.Format(
-                        LocalizableStrings.InstallationSucceeded,
-                        string.Join(", ", package.Commands.Select(c => c.Name)),
-                        package.Id,
-                        package.Version.ToNormalizedString()).Green());
+                string.Format(
+                    LocalizableStrings.InstallationSucceeded,
+                    string.Join(", ", package.Commands.Select(c => c.Name)),
+                    package.Id,
+                    package.Version.ToNormalizedString()).Green());
```

Comment:
```
if !(_verbosity.IsQuiet())
{
        _reporter.WriteLine(
        string.Format(
            LocalizableStrings.InstallationSucceeded,
            string.Join(", ", package.Commands.Select(c => c.Name)),
            package.Id,
            package.Version.ToNormalizedString()).Green());
}
return 0;
```

Reply:
The affirmative condition might be a better choice. Thanks for pointing it out!

Reply:
I like the `if (!_verbosity.IsQuiet())` option better, too. Nice to have just one exit point.

Reply:
And also (tiny nit) it'd be nice to have one blank line on either side of the if block.